src/span.js

```diff
@@ -144,6 +144,17 @@ export default class Span {
      **/
     setOperationName(operationName: string): Span {
         this._operationName = operationName;
+        // If the span has debug set, then the question of resampling
+        // after setting the operation name is irrelevant.  The span is always
+        // sampled.
```

Comment:
I would like to consider it user error to call setOperationName more than once.  What is a logical reason for why someone would need to call it more than once?